biom/table.py

```diff
@@ -3343,6 +3353,14 @@ def to_hdf5(self, h5grp, generated_by, compress=True):
         compress : bool, optional
             Defaults to ``True`` means fields will be compressed with gzip,
             ``False`` means no compression
+        format_fs : dict, optional
+            Specify custom formatting functions for metadata fields. This dict
+            is expected to be {'metadata_field': function}, where the function
+            signature is (h5py.Group, str, dict, bool) corresponding to the
+            specific group the metadata dataset will be associated with, the
```

Comment:
"group" is slightly ambiguous in this context; maybe add "HDF5 group"?  I do recognize that the preceding signature has it listed as `h5py.Group`, but it still could be slightly confusing, IMHO.